docs/md/installing.md

```diff
@@ -18,22 +18,23 @@ To install JFlex on Windows, follow these three steps:
         +--doc\                        (FAQ and manual) 
         +--examples\ 
             +--byaccj\                 (calculator example for BYacc/J) 
-            +--cup\                    (calculator example for cup) 
+            +--cup-maven\              (calculator example for cup and maven) 
             +--interpreter\            (interpreter example for cup) 
             +--java\                   (Java lexer specification) 
             +--simple-maven\           (example scanner built with maven) 
             +--standalone-maven\       (a simple standalone scanner, 
                                         built with maven) 
             +--zero-reader\            (Readers that return 0 characters) 
-        +--lib\                        (precompiled classes, skeleton files) 
+        +--lib\                        (precompiled classes) 
         +--src\ 
             +--main\ 
+                +--config\             (PMD source analyzer configuration) 
                 +--cup\                (JFlex parser spec) 
                 +--java\ 
-                    +--java_cup\ 
-                        +--runtime\    (source code of cup runtime classes) 
                     +--jflex\          (source code of JFlex) 
+                        +--anttask\    (source code of JFlex Ant Task) 
```

Comment:
Hmm, sorry, I completely forgot about this section of the documentation. Maybe the documentation should only have the examples, though.

Reply:
No worries. It's worth looking at this again and seeing if we can delete the whole dir structure thing. 20 years ago I put it in to reassure people they have unpacked the archive correctly, but you'd think that such things are more routine these days. Since I've already updated it, though, let's do that for the next release :-)

Reply:
Yes. One thing that could be nice in the future is to have one README.md per directory, that explains the content of the directory.
But LGTM for now.